test/Constraints/optional.swift

```diff
@@ -357,3 +357,16 @@ struct S {
     }
   }
 }
+
+func rdar_53238058() {
```

Comment:
Can we add a note stating what the original problem was for those of us who can't look at the radar similar to the above:
```
// rdar://problem/47776586 - Diagnostic refers to '&' which is not present in the source code
```

Reply:
Ah sure, I'll do that, thanks!

Reply:
Done!